pom-eranian/pokerogue
# [Sprite] 774-Minior spritesheet fixes

all minion cores now have identical color indexing, spritesheet framing and json animation.

[ [Bug][Anim] Minior's animation wraps around to the left #4130 ](https://github.com/pagefaultgames/pokerogue/issues/4130)


### Screenshots/Videos

https://github.com/user-attachments/assets/a11d4a49-428f-42e8-acfd-59754dcfe97a



## How to test the changes?
set for player and opponent
species: minior
form: increment to see each color
shiny: set as needed
**has no variants**

**to rid the shield quickly**
level: 20
moveset: dragon rage x4


## Checklist
~~- [ ] Have I considered writing automated tests for the issue?~~
~~- [ ] If I have text, did I make it translatable and add a key in the English locale file(s)?~~
